types/react-twitter-conversion-tracker/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "allowSyntheticDefaultImports": true
```

```suggestion
        "forceConsistentCasingInFileNames": true
```
just like in comment on other, it's not reuqired here, shoudl work with defalt config created with templates.

---

Yup just tried this out with a blank CRA project created from template, it works. Thanks.